pre-commit.sh

```diff
@@ -4,9 +4,9 @@ branch="$(git rev-parse --abbrev-ref HEAD)"
 
 directories="$(git diff --staged --name-status --diff-filter=DR)"
 
-# Prevent direct commit to master branch
```

Comment:
Funny -- I wonder if this repo was created before this functionality was added directly to GitHub.